### PR TITLE
Fix file size calculation for var itemSizeBytes

### PR DESCRIPTION
**Pull request by tsvietOK**
This var hasn't been changed in PR #708

### CODE REVIEW

File: Files/View Models/ItemViewModel.cs [hunk 1]
```diff
@@ -848,28 +848,27 @@ private void AddFile(WIN32_FIND_DATA findData, string pathRoot)
                 systemTimeOutput.Second,
                 systemTimeOutput.Milliseconds);
             long fDataFSize = findData.nFileSizeLow;
-            long fileSize;
+            long itemSizeBytes;
             if (fDataFSize < 0 && findData.nFileSizeHigh > 0)
             {
-                fileSize = fDataFSize + 4294967296 + (findData.nFileSizeHigh * 4294967296);
+                itemSizeBytes = fDataFSize + 4294967296 + (findData.nFileSizeHigh * 4294967296);
```
Comment (duke7553) [inline Files/View Models/ItemViewModel.cs:854]: What is the significance of the 4294967296 value applied here? Shouldn't we initially store this value as a constant?

Comment (tsvietOK) [reply Files/View Models/ItemViewModel.cs:854]: Of course we can use this value as constant. The main algorithm taken from [here](http://www.freevbcode.com/ShowCode.asp?ID=6836)

Comment (duke7553) [reply Files/View Models/ItemViewModel.cs:854]: Good. Let me know when you've made that change

Comment (tsvietOK) [reply Files/View Models/ItemViewModel.cs]: @duke7553 Done.

**Comment by tsvietOK** [comment]
May be in the future we should do global code align, i mean move all const to top of file, check file names for casing, and check fields, constructors and methods order according to code convention

**Pull request merged**